packages/editor-ui/src/components/PersonalizationModal.vue

```diff
@@ -758,9 +758,9 @@ export default defineComponent({
 			) {
 				const onboardingResponse = await this.uiStore.getNextOnboardingPrompt();
 				const promptTimeout =
```

Comment:
Maybe just exit fast if we don't get an response? Wouldn't need to check if `onboardingResponse` is defined with every access

```ts
if (!onboardingResponse) {
  return
}
...
```